src/common/sentry.ts

```diff
@@ -21,95 +21,78 @@
 
 import { CaptureConsole, Dedupe, Offline } from "@sentry/integrations";
 import type { Event as SentryEvent } from "@sentry/types";
-import { sentryDsn, sentryDsnIsValid, isProduction } from "./vars";
+import { sentryDsn, isProduction } from "./vars";
 import { UserStore } from "./user-store";
 import logger from "../main/logger";
 
 // https://www.electronjs.org/docs/api/process#processtype-readonly
-const electronProcessType = ["browser", "renderer", "worker"] as const;
+type ElectronProcessType = "browser" | "renderer" | "worker";
+
+const processType = process.type as ElectronProcessType;
 
 export const integrations = [
   new CaptureConsole({ levels: ["error"] }),
   new Dedupe(),
   new Offline()
 ];
 
-const initialScope = (processType: typeof electronProcessType[number]) => {
-  return {
-    tags: {
-      // "translate" browser to 'main' as Lens developer more familiar with the term 'main'
-      "process": processType === "browser" ? "main" : "renderer"
-    }
-  };
+const initialScope = {
+  tags: {
+    // "translate" browser to 'main' as Lens developer more familiar with the term 'main'
+    "process": processType === "browser" ? "main" : "renderer"
+  }
 };
 
-export const environment = isProduction ? "production" : "development";
+const environment = isProduction ? "production" : "development";
 
-const logInitError = (reason: string) => {
+function logInitError(reason: string) {
   logger.warn(`⚠️  [SENTRY-INIT]: ${reason}, Sentry is not initialized.`);
-};
-
-export const beforeSend = (event: SentryEvent) => {
-  const allowErrorReporting = UserStore.getInstance().allowErrorReporting;
+}
 
-  if (allowErrorReporting) return event;
+function beforeSend(event: SentryEvent) {
+  if (UserStore.getInstance().allowErrorReporting) {
+    return event;
+  }
 
-  logger.info(`🔒  [SENTRY-BEFORE-SEND-HOOK]: allowErrorReporting: ${allowErrorReporting}. Sentry event is caught but not sent to server.`);
+  logger.info(`🔒  [SENTRY-BEFORE-SEND-HOOK]: allowErrorReporting: false. Sentry event is caught but not sent to server.`);
   logger.info("🔒  [SENTRY-BEFORE-SEND-HOOK]: === START OF SENTRY EVENT ===");
   logger.info(event);
   logger.info("🔒  [SENTRY-BEFORE-SEND-HOOK]: ===  END OF SENTRY EVENT  ===");
 
   // if return null, the event won't be sent
   // ref https://github.com/getsentry/sentry-javascript/issues/2039
   return null;
-};
-
-export const SentryInit = () => {
-  if (!sentryDsnIsValid) {
-    logInitError("Sentry DSN is not valid");
+}
 
-    return;
+async  function requireSentry() {
```

Comment:
Seems to be an extra space?
```suggestion
async function requireSentry() {
```